lib/init.php

```diff
@@ -57,3 +57,58 @@ function gutenberg_menu() {
 	);
 }
 add_action( 'admin_menu', 'gutenberg_menu', 9 );
+
+function disable_scripts() {
+	// disable loading and enqueuing block editor scripts and styles
+	// can't be `__return_false` because this function is already registered for this filter
```

Comment:
Perhaps if we add it with a different priority (the third `add_filter()` argument)?